File: Android_Java/wit-example-ble5/app/src/main/java/com/wit/witsdk/sensor/modular/device/entity/CalcOption.java
package com.wit.witsdk.sensor.modular.device.entity;

/**
 * 数据解算配置类
 *
 * @author huangyajun
 * @date 2022/5/20 9:50
 */
public class CalcOption {

    /**
     * 名称
     */
    public String name;

    /**
     * 名称
     */
    public String key;

    /**
     * 名称
     */
    public String csScript;

    /**
     * 名称
     */
    public String suffix;

    /**
     * 名称
     */
    public String enableOffset;

    /**
     * 名称
     */
    public int sort;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getCsScript() {
        return csScript;
    }

    public void setCsScript(String csScript) {
        this.csScript = csScript;
    }

    public String getSuffix() {
        return suffix;
    }

    public void setSuffix(String suffix) {
        this.suffix = suffix;
    }

    public String getEnableOffset() {
        return enableOffset;
    }

    public void setEnableOffset(String enableOffset) {
        this.enableOffset = enableOffset;
    }

    public int getSort() {
        return sort;
    }

    public void setSort(int sort) {
        this.sort = sort;
    }
}

File: Unity_C#/Android/Assets/Lib/Device/Device/Entity/CalcOption.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml.Serialization;
using Wit;
using Wit.SDK;
using Wit.SDK.Modular;
using Wit.SDK.Modular.Sensor;
using Wit.SDK.Sensor.Device.Enum;

namespace Wit.SDK.Modular.Sensor.Device.Entity
{


    /// <summary>
    /// 数据解算配置
    /// </summary>
    public class CalcOption
    {

        /// <summary>
        /// 名称
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// 数据key
        /// </summary>
        public string Key { get; set; }

        /// <summary>
        /// 数据key
        /// </summary>
        public string DescribeKey { get; set; }

        /// <summary>
        /// c#脚本
        /// </summary>
        public string CsScript { get; set; }

        /// <summary>
        /// 后缀
        /// </summary>
        public string Suffix { get; set; }

        /// <summary>
        /// 是否开启零偏
        /// </summary>
        public bool EnableOffset { get; set; }

        /// <summary>
        /// 排序
        /// </summary>
        public int Sort { get; set; }

        /// <summary>
        /// 数据类型
        /// </summary>
        public DataType DataType { get; set; } = DataType.String;
    }

}
